src/ape/contracts/base.py

```diff
@@ -2,14 +2,13 @@
 
 from eth_utils import to_bytes
 
+from ape.api.address import Address, AddressAPI
+from ape.api.base import dataclass
+from ape.api.providers import ProviderAPI, ReceiptAPI, TransactionAPI
```

Comment:
These and `Address`, `AddressAPI` should be exported out of `ape.api`

Reply:
Same with the `ape.api.providers` ?

Reply:
yes!